converter/shieldhit/geo.py

```diff
@@ -4,6 +4,8 @@
 from math import log10, ceil, isclose
 from scipy.spatial.transform import Rotation as R
 
+PREDEVINED_MATERIALS = [0, 1000]
```

Comment:
Add a meaningful name for 0 and 1000. It is not clear now which one is vacuum and which - black hole